tests/system_test.py

```diff
@@ -173,19 +173,37 @@ def get_local_host_socket(socket_address_family='IPv4'):
     return s, host
 
 
-def port_available(port, socket_address_family='IPv4'):
+def port_refuses_connection(port, socket_address_family='IPv4'):
```

Comment:
I would call this function -  check_port_refuses_connection

Reply:
I did not go with `check_port_permits_connection` in the end, because the purpose of the function is to check for `errno.ECONNREFUSED`.